frontend/tsconfig.tsbuildinfo

```diff
@@ -0,0 +1 @@
[101,1],[102,1],[103,1],[104,1],[105,1],[106,1],[107,1],[108,1],[109,1],[110,1],[111,1],[51,1],[115,1],[334,1],[335,1],[336,1],[338,1],[337,1],[339,1],[340,1],[344,1],[342,1],[343,1],[345,1],[346,1],[347,1],[348,1],[349,1],[350,1],[352,1],[341,1],[357,1],[353,1],[355,1],[354,1],[351,1],[356,1],[332,1],[358,1],[333,1],[213,1],[365,1],[49,1],[53,1],[457,1],[52,1],[458,1],[459,1],[460,1],[194,1],[209,1],[50,1],[246,1],[245,1],[247,1],[406,1],[215,1],[59,1],[169,1],[171,1],[129,1],[139,1],[130,1],[131,1],[140,1],[120,1],[180,1],[182,1],[164,1],[163,1],[162,1],[185,1],[161,1],[188,1],[116,1],[118,1],[168,1],[172,1],[176,1],[123,1],[122,1],[143,1],[151,1],[126,1],[121,1],[117,1],[155,1],[154,1],[144,1],[114,1],[152,1],[145,1],[153,1],[156,1],[132,1],[146,1],[127,1],[150,1],[149,1],[128,1],[142,1],[141,1],[133,1],[134,1],[136,1],[135,1],[137,1],[187,1],[138,1],[58,1],[166,1],[174,1],[178,1],[158,1],[119,1],[160,1],[165,1],[148,1],[147,1],[125,1],[124,1],[170,1],[167,1],[48,1],[57,1],[54,1],[55,1],[56,1],[173,1],[175,1],[177,1],[179,1],[192,1],[181,1],[191,1],[183,1],[184,1],[186,1],[189,1],[190,1],[157,1],[198,1],[248,1],[443,1],[442,1],[217,1],[216,1],[197,1],[196,1],[195,1],[240,1],[241,1],[239,1],[238,1],[2,1],[3,1],[4,1],[5,1],[6,1],[7,1],[8,1],[9,1],[203,1],[330,1],[331,1],[364,1],[432,1],[369,1],[452,1],[436,1],[375,1],[415,1],[419,1],[423,1],[427,1],[429,1],[430,1],[328,1],[237,1],[244,1],[243,1],[242,1],[232,1],[326,1],[320,1],[322,1],[321,1],[316,1],[319,1],[318,1],[317,1],[252,1],[255,1],[254,1],[253,1],[251,1],[259,1],[258,1],[257,1],[256,1],[219,1],[231,1],[230,1],[309,1],[312,1],[311,1],[310,1],[267,1],[270,1],[269,1],[268,1],[235,1],[306,1],[308,1],[307,1],[325,1],[324,1],[323,1],[218,1],[305,1],[304,1],[206,1],[315,1],[314,1],[313,1],[205,1],[236,1],[329,1],[200,1],[220,1],[234,1],[327,1],[229,1],[207,1],[208,1],[204,1],[280,1],[233,1],[202,1],[199,1],[250,1],[211,1],[226,1],[214,1],[221,1],[461,1],[210,1],[225,1],[224,1],[462,1],[228,1],[227,1],[201,1],[273,1],[265,1],[463,1],[266,1],[271,1],[464,1],[272,1],[274,1],[465,1],[275,1],[276,1],[281,1],[389,1],[390,1],[416,1],[417,1],[418,1],[260,1],[447,1],[448,1],[440,1],[441,1],[437,1],[444,1],[445,1],[446,1],[451,1],[449,1],[450,1],[438,1],[439,1],[435,1],[433,1],[434,1],[285,1],[286,1],[287,1],[290,1],[284,1],[282,1],[283,1],[288,1],[289,1],[388,1],[386,1],[387,1],[385,1],[384,1],[380,1],[382,1],[381,1],[379,1],[383,1],[378,1],[374,1],[372,1],[373,1],[278,1],[299,1],[297,1],[298,1],[400,1],[466,1],[401,1],[402,1],[414,1],[412,1],[413,1],[405,1],[403,1],[467,1],[404,1],[411,1],[408,1],[407,1],[409,1],[410,1],[399,1],[397,1],[395,1],[396,1],[398,1],[296,1],[293,1],[291,1],[292,1],[294,1],[295,1],[422,1],[420,1],[421,1],[394,1],[392,1],[393,1],[303,1],[301,1],[302,1],[426,1],[424,1],[425,1],[279,1],[362,1],[359,1],[360,1],[361,1],[376,1],[431,1],[377,1],[363,1],[428,1],[391,1],[371,1],[300,1],[277,1],[249,1],[261,1],[262,1],[263,1],[264,1],[370,1],[367,1],[368,1],[366,1]]},"version":"4.5.2"}
```

Comment:
I think this file can be added to `.gitignore`: https://github.com/vercel/next.js/discussions/30815

Reply:
you are right, on it